packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -54,6 +58,16 @@ export interface PresentationInstanceFilterDialogProps {
   initialFilter?: PresentationInstanceFilterInfo;
 }
 
+/**
+ * Set of custom handlers that will additionally be called on each action respectively.
+ * @beta
+ */
+export interface FilteringDialogToolbarHandlers {
+  handleApply: () => void;
+  handleClose?: () => void;
```

Comment:
These don't need to be optional. They are provided by `PresentationInstanceFilterDialog` and are always defined. (Atleast `handleApply` and `handleReset`). In case of `handleClose` I think `PresentationInstanceFilterDialog` could pass something like `() => { props.onClose && props.onClose() }`. This way API for customizing buttons will be cleaner for consumer as they won't need to make undefined checks.